packages/components/src/internal/components/assay/AssayWizardModel.ts

```diff
@@ -16,6 +16,7 @@ export interface AssayUploadOptions extends AssayDOM.ImportRunOptions {
     dataRows?: any; // Array<any>
     maxFileSize?: number;
     maxRowCount?: number;
+    resultsFiles?: any[];
```

Comment:
Follow-on from the comment in `@labkey/api` PR that this could/should be `File[]`.